aws-mfa

```diff
@@ -198,15 +201,19 @@ def validate(args, config):
                     % (diff.total_seconds(), exp))
 
 
+def log_error_and_exit(message):
+    """Log an error message and exit with error"""
+    logger.error(message)
+    sys.exit(1)
+
+
 def get_credentials(short_term_name, lt_key_id, lt_access_key, args, config):
     try:
         token_input = raw_input
     except NameError:
         token_input = input
 
-    mfa_token = token_input('Enter AWS MFA code for device [%s] '
-                            '(renewing for %s seconds):' %
-                            (args.device, args.duration))
+    mfa_token = token_input('Enter AWS MFA code for device [%s] ' % args.device)
```

Comment:
Why are we removing the second information? We're already providing input in seconds, so I find it useful.